### PR TITLE
{bio}[intel/2017a] HISAT2 2.1.0 (REVIEW)

### PR DESCRIPTION
**Pull request by smoors**
{bio}[intel/2017a] HISAT2 2.1.0 (REVIEW)

### CODE REVIEW

File: easybuild/easyconfigs/h/HISAT2/HISAT2-2.1.0_intel-fixes.patch [hunk 1]
```diff
@@ -0,0 +1,13 @@
+--- hisat2-2.1.0/processor_support.h.orig	2017-10-05 16:50:19.290831227 +0200
```
Comment (boegel) [inline easybuild/easyconfigs/h/HISAT2/HISAT2-2.1.0_intel-fixes.patch]: @smoors Please include a brief description of the patch + mention author or include reference to upstream source of patch

Comment (smoors) [reply easybuild/easyconfigs/h/HISAT2/HISAT2-2.1.0_intel-fixes.patch]: sorry for not including the author (you) :)

File: easybuild/easyconfigs/h/HISAT2/HISAT2-2.1.0-intel-2017a.eb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+ (both DNA and RNA) against the general human population (as well as against a single reference genome)."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+source_urls = ['ftp://ftp.ccb.jhu.edu/pub/infphilo/%(namelower)s/downloads/']
+sources = ['%(namelower)s-%(version)s-source.zip']
+patches = ['HISAT2-%(version)s_intel-fixes.patch']
+checksums = [
+    '89a276eed1fc07414b1601947bc9466bdeb50e8f148ad42074186fe39a1ee781',  # hisat2-2.1.0-source.zip
+    'd74a8ee4cc706b7ec1d9f48ca393fa2233f9d9f7988bd925be004532754a6174',  # HISAT2-2.1.0_intel-fixes.patch
```
Comment (boegel) [inline easybuild/easyconfigs/h/HISAT2/HISAT2-2.1.0-intel-2017a.eb]: @smoors checksum should now be `7986dbcdb56b4bf5494dd70b215e90e321ca887dbdafc48de7a13e6b5abd038b` after adding the description/author?

Comment (smoors) [reply easybuild/easyconfigs/h/HISAT2/HISAT2-2.1.0-intel-2017a.eb]: fixed.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2430.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/ff15479cde50fbe2db6425f793d63134 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2092.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/a4d207630932860063291d029c1648da for a full test report.

**Comment by boegel** [comment]
Going in, thanks @smoors!

**Pull request merged**